qpc_tools/utils.py

```diff
@@ -170,15 +170,22 @@ def get_password(args_dictionary):
     :param args_dictionary: the dictionary containing the args and values
     :returns: the dictionary with updated passwords
     """
-    arg_prompt = OrderedDict([
-        ('rh_registry_username', 'Enter registry.redhat.io username: '),
-        ('rh_registry_password', 'Enter registry.redhat.io password: '),
-        ('server_password', 'Enter server password: '),
-        ('db_password', 'Enter database password: ')
-    ])
+    if args_dictionary.get('registry_no_auth'):
+        arg_prompt = OrderedDict([
+            ('server_password', 'Enter server password: '),
+            ('db_password', 'Enter database password: ')
+        ])
+    else:
+        registry_url = args_dictionary.get('registry_url', 'registry url')
```

Comment:
if you must set a default here, i would make it `registry.redhat.io` instead of `registry url`. But ideally this value should always be there since we do default it in the argparser to `registry.redhat.io` and if it isn't there -errors will occur later.